idl/partition_checksum.idl.hh

```diff
@@ -154,3 +154,13 @@ struct repair_update_system_table_request {
 
 struct repair_update_system_table_response {
 };
+
```

Comment:
changelog should explain why flushing hints and batchlog is good.

Reply:
Sure.